### PR TITLE
Add transitions to title area

### PR DESCRIPTION
**Pull request by rak-phillip**
This adds transitions to the title area in an attempt to make navigating between Images and Add Image a little more pleasing.

Signed-off-by: Phillip Rak <rak.phillip@gmail.com>

### CODE REVIEW

**Comment by rak-phillip** [comment]
### Transitions applied (Title slides when back button appears and add button fades in and out)

Kapture 2021-12-08 at 15 35 05

File: src/layouts/default.vue [hunk 1]
```diff
@@ -5,25 +5,42 @@
...
-            {{ title }}
-          </h1>
-          <section
-            v-if="action"
-            class="actions"
+            <button
+              v-if="isChild"
+              key="back-btn"
+              class="btn role-link btn-sm btn-back fade-group-item"
+              type="button"
```
Comment (evertonlperes) [inline src/layouts/default.vue:17]: Can we add a test element `data-test="back-btn"`?
It can be handy to test on E2E specs :D

**Comment by evertonlperes** [comment]
Minor changes for helping E2E tests with test-tags and nicely done! Really enjoyed the transition feature!

File: src/layouts/default.vue (hunk 1 above)
Comment (rak-phillip) [reply src/layouts/default.vue:18]: Sounds easy enough, thanks for calling this out!

**Pull request merged**